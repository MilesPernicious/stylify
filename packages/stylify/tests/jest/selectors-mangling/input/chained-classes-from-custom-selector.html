<!--
stylify-components
  container: `
    width:1px
    &--lg { width:2px }
  `
/stylify-components
-->
<div class="container container--lg"></div>
