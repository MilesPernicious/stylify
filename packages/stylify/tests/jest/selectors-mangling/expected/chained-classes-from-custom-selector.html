<!--
stylify-components
  container: `
    width:1px
    &--lg { width:2px }
  `
/stylify-components
-->
<div class="a b"></div>
